docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-installer.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Upgrade YugabyteDB Anywhere using YBA Installer
+headerTitle: Upgrade YugabyteDB Anywhere
+linkTitle: Upgrade installation
+description: Use YBA Installer to upgrade YugabyteDB Anywhere
+menu:
+  preview_yugabyte-platform:
+    identifier: upgrade-yp-1-installer
+    parent: upgrade
+    weight: 80
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li>
+    <a href="../upgrade-yp-installer/" class="nav-link active">
+      <i class="fa-solid fa-building"></i>YBA Installer</a>
+  </li>
+
+  <li>
+    <a href="../upgrade-yp-replicated/" class="nav-link">
+      <i class="fa-solid fa-cloud"></i>Replicated</a>
+  </li>
+
+  <li>
+    <a href="../upgrade-yp-kubernetes/" class="nav-link">
+      <i class="fa-regular fa-dharmachakra" aria-hidden="true"></i>Kubernetes</a>
+  </li>
+
+</ul>
+
+To upgrade using YBA Installer, first download the version of YBA Installer corresponding to the version of YBA you want to upgrade to.
+
+Download and extract the YBA Installer by entering the following commands:
+
+```sh
+$ wget https://downloads.yugabyte.com/releases/{{<yb-version version="preview" format="long">}}/yba_installer_full-{{<yb-version version="preview" format="build">}}-linux-x86_64.tar.gz
+$ tar -xf yba_installer_full-{{<yb-version version="preview" format="build">}}-linux-x86_64.tar.gz
+$ cd yba_installer_full-{{<yb-version version="preview" format="build">}}/
+```
+
+When ready to upgrade, run the `upgrade` command from the untarred directory of the target version of the YBA upgrade:
+
+```sh
+$ sudo ./yba-ctl upgrade
+```
+
+The upgrade takes a few minutes to complete. When finished, use the status command to verify that YBA has been upgraded to the target version:
```

Comment:
In case of errors, we leave around the previous version software to make it easy to switch back. If there are any issues with upgrade they should immediately contact support as we don't have any rollback mechanism built into the product.

Reply:
So it is there, but they shouldn't do anything with the old version, just contact support if they want to roll back?
Should we mention how to get rid of it after they are sure about success of upgrade, or anything like that.

Reply:
Yeah, we maybe don't even want to tell them it's there. We will GC anything older than the most recent version so at one time the most a customer could see is two versions in the /opt/yugabyte/software directory. But, they are not advised to switch to the old software dir without supports help. Technically if the upgrade doesn't complete we shouldn't swap over to the new software so when they navigate to the running YBA after a failed upgrade they should still see the old version, but in case there are unforeseen issues it would be a support call if upgrade fails and they appear to be upgraded. If upgrade fails and they are still the same version then it's probably they need to fix some issue on the node and rerun upgrade.

Reply:
OK I added a note to contact support if there are problems and left it at that